### ISSUE TITLE
Very laggy in Chrome. 

### ISSUE THREAD

**Issue by theoryshaw** [benweet/stackedit#1770]
Unusually laggy in Chrome.  No lag in firefox.

Is there a screen capture in 'Developer Tools', that I can do, to help you troubleshoot?

**Comment by Emnolope** [comment]
I am also experiencing severe lag issues. I am using Android Brave with chromium 97.0.4792.71. What is your chromium version?

**Comment by Emnolope** [comment]
> Unusually laggy in Chrome. No lag in firefox.
> 
> Is there a screen capture in 'Developer Tools', that I can do, to help you troubleshoot?

Perhaps we can work together to solve?

**Comment by theoryshaw** [comment]
i haven't experienced lag recently.  
Google Chrome
Version 97.0.4692.99 (Official Build) (64-bit)

will report again, if it happens again.

**Comment by Emnolope** [comment]
Gonna make a new issue, since (I think?) Desktop Chromium is different than mobile chromium.

**Comment by GrimSoftworks** [comment]
I don't believe its lag, but certain things cause the markdown editor to preview "slowly". I think this is due to the way we are editing.

I noticed it most when attempting to make markdown tables with copy/pasted information. I don't think its the editor that's the problem but a user generated issue with too much input
